### DISCUSSION TITLE
can not find EthereumRlp in swift

### DISCUSSION THREAD

**Discussion by lovivid** [trustwallet/wallet-core#3587] [Q&A]
**Describe the bug**
read this code https://github.com/trustwallet/wallet-core/blob/master/swift/Tests/Blockchains/EthereumRlpTests.swift 
cannot find 
EthereumRlpRlpList
EthereumRlpRlpItem
EthereumRlpEncodingInput
EthereumRlp
EthereumRlpEncodingOutput

**To Reproduce**

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by satoshiotomakan** [comment]
Hi @lovivid, could you tell how do you run the tests? Do you run `tools/generate-files` before running iOS tests?
